protos/sources/vega/assets.proto

```diff
@@ -66,6 +66,8 @@ message ERC20 {
   // There’s no limit on the size of a withdrawal
   // note: this is a temporary measure that can be changed by governance.
   string withdraw_threshold = 3;
+  // Blockchain ID the asset originated from.
```

Comment:
does this also need to be exposed in graphql?

Reply:
API will live in another PR, in a distant future.